.github/ISSUE_TEMPLATE/dm-support.md

```diff
@@ -50,16 +50,16 @@ Additionally, please provide the following info before submitting your issue. Th
 
 3. Operation logs
 
-    - [ ] Please upload `dm-worker.log` for every dm-worker instance if possible.
+    - [ ] Please upload `dm-worker.log` for every DM-worker instance if possible.
     - [ ] Please upload `dm-master.log` if possible.
     - [ ] Other interesting logs.
     - [ ] Output of dmctl's commands with problems.
 
 4. Configuration of the cluster and the task
 
-    - [ ] `dm-worker.toml` for every dm-worker instance if possible.
-    - [ ] `dm-master.toml` for dm-master if possible.
+    - [ ] `dm-worker.toml` for every DM-worker instance if possible.
+    - [ ] `dm-master.toml` for DM-master if possible.
     - [ ] task config, like `task.yaml` if possible.
-    - [ ] `inventory.ini` if deployed by dm-ansible.
+    - [ ] `inventory.ini` if deployed by DM-Ansible.
```

Comment:
also update `DM-Ansible` in `How did you deploy DM: manually, or via dm-ansible?`

and more `DM-Ansible` (not in code blocks) need to update in `dm-ansible.md`